docs/user-guide/reference/autoscaling.md

```diff
@@ -53,8 +53,8 @@ spec:
   start processing, because the autoscaling algorithm will divide the TPS by the number of pods even if the pod is not `Running`.
 - `scaleDownCooldownSeconds` - After a scaling operation, how many seconds to wait for the same vertex, if the follow-up
   operation is a scaling down, defaults to `90`.
-- `zeroReplicaSleepSeconds` - How many seconds it will wait after scaling down to `0`, defaults to `120`.
-  Numaflow autoscaler periodically scales up a vertex pod to "peek" the incoming data, this is the period of time to wait before peeking.
+- `zeroReplicaSleepSeconds` - How many seconds it will wait after scaling a source vertex replicas down to `0`, defaults to `120`.
```

Comment:
Please also update the API [here](https://github.com/numaproj/numaflow/blob/ee36df51040ad2a71ac513aa196a126d82381ede/pkg/apis/numaflow/v1alpha1/vertex_types.go#L561). After making the change, please re-build the project which will automatically update the API descriptions, website etc. accordingly.

Reply:
updated and ran  `make pre-push -b` to generate docs and api specs